tests/vcs/git/GitCommitBearTest.py

```diff
@@ -81,6 +81,11 @@ def test_check_prerequisites(self):
         finally:
             shutil.which = _shutil_which
 
+    def test_get_metadata(self):
+        self.assertEqual(
+            GitCommitBear.get_metadata().name,
+            "<Merged signature of 'run', 'check_shortlog', 'check_body'>")
```

Comment:
test params too please, I think this implementation doesn't get stuff from run correctly